gulp/tasks/gzip.js

```diff
@@ -2,12 +2,16 @@
 
 var gulp   = require('gulp');
 var gzip   = require('gulp-gzip');
+var size   = require('gulp-size');
+var gulpif = require('gulp-if');
 var config = require('../config');
 
 gulp.task('gzip', function() {
 
   return gulp.src(config.gzip.src)
+  	.pipe(gulpif(config.logsizes, size({showFiles: true, title:'Pre-gzip'})))
```

Comment:
Could you fix the indentation here to match the surrounding lines, and also put the options for `size` on separate lines as seen in other places?